### DISCUSSION TITLE
Chromium -> Homebrew for MacOS

### DISCUSSION THREAD

**Discussion by GaryElshaw** [Eloston/ungoogled-chromium#1801] [General]
Is there any reason why the latest version hasn't hit Homebrew yet?

**Comment by networkException** [comment]
Different distributions get maintained by different people in their free time.

Also, the homebrew release is very much up to date: https://formulae.brew.sh/cask/eloston-chromium

**Comment by GaryElshaw** [comment]
Maybe i'm misunderstanding, and no shade. My current version is 96.0.4664.110, this last week was updated to be 97.0.4692.71 and i was just wondering if there is something going on with the build i'm not seeing. Wasn't trying to be a dick, honest.

**Reply by networkException** [reply]
Sorry if I sounded mean, that wasn't my intention at all. I hope you can get everything updated :)

**Comment by kramred** [comment]
Hello,
`97.0.4692.71` should be the latest version for now. I had some trouble with GitHub Actions failing repeatedly for the arm64 builds so I changed the format of the release tag to allow builds started in parallel to finish with a release that is not blocking the other builds (except if they finish on the exact same second in time, where I estimated the probability to be low enough to take that risk)
So maybe the maintainer of the brew cask code (who is not me) had to make some adjustments too.

**Reply by GaryElshaw** [reply]
I just tried a brew reinstall, and 96 was reinstalled, but 97 doesn't appear to be an option.

**Reply by kramred** [reply]
try this:
```bash
brew update
brew info --cask eloston-chromium
```
check what version it is showing, for me it's `97.0.4692.71-1.2_x86-64` – are you using an Intel oder Apple M1 CPU?

**Reply by GaryElshaw** [reply]
Still 96 :-(

![Screenshot 2022-01-14 at 4 04 02 AM](https://user-images.githubusercontent.com/69029666/149355081-7a639ef3-ceb9-43a4-bf8a-6deac2ed5879.png)

**Reply by kramred** [reply]
This may be related to the error messages from brew. See the [cask code](https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/eloston-chromium.rb) on github, which shows the current version.

I think you have to fix your brew issues/errors first then it should work.

You can also download and install the current version [manually](https://github.com/kramred/ungoogled-chromium-macos/releases) till you've done that.

**Reply by GaryElshaw** [reply]
Thank you, I think you just verified an uninstall and reinstall of homebrew is necessary. I've tried 'brew doctor' but the complaints are largely python related, which i was expecting, but not the casks. Thanks again.

**Reply by GaryElshaw** [reply]
Uninstalled, reinstalled Homebrew, and still the same version. Interestingly, the errors are still there.

Running `brew update --preinstall`...
error: feeding unmodified Casks/stand.rb to diffcore
error: cannot rebase: Your index contains uncommitted changes.
error: Please commit or stash them.

**Reply by kramred** [reply]
Try asking in the brew community about that error, they may be of more use in finding a way to fix this.

**Reply by GaryElshaw** [reply]
Thanks!
